jdaviz/components/viewer_data_select_item.vue

```diff
@@ -122,7 +159,7 @@ module.exports = {
         } else if (this.$props.viewer.reference === 'mask-viewer') {
           return ['MASK', 'DQ'].indexOf(extension) !== -1
         } else if (this.$props.viewer.reference === 'spectrum-viewer') {
-          return ['SCI', 'FLUX'].indexOf(extension) !== -1
+          return this.$props.item.name.indexOf('[FLUX]') === -1
```

Comment:
This change seems unrelated to Imviz and reverts the behavior added in #2409 . Is this intentional or another rebase error?

Reply:
I believe this is a rebase error. Checking now.